data/json/recipes/armor/suit.json

```
@@ -809,7 +809,7 @@
     "subcategory": "CSC_ARMOR_SUIT",
     "skill_used": "tailor",
     "difficulty": 3,
-    "time": "10 h",
+    "time": "40 m",
```

```suggestion
    "time": "2 h",
```
I am going to be generous for you @RoyalFox2140